### PR TITLE
Prow bump: add an auto label to distinguish it from other bumps

### PR DESCRIPTION
**Pull request by chaodaiG**
In preparing for tide auto-merging prow autobump PRs

### CODE REVIEW

File: config/prow/autobump-config/prow-component-autobump-config.yaml [hunk 1]
```diff
@@ -14,6 +14,8 @@ includedConfigPaths:
 excludedConfigPaths:
   - "config/prow-staging"
 targetVersion: "latest"
+labels:
+  - "auto/trusted-pullrequest" # This label is used by tide for identifying trusted PR
```
Comment (cjwagner) [inline config/prow/autobump-config/prow-component-autobump-config.yaml:18]: Please also create this label via the label_sync [config](https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml).

Comment (chaodaiG) [reply config/prow/autobump-config/prow-component-autobump-config.yaml:18]: Done

**Review by cjwagner — APPROVED** [review]

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm
Remove hold when ready

**Pull request merged**